src/users/users.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Body, Controller, Get, Param, Post } from '@nestjs/common';
+import { CreateUsersDto } from './DTO/create-user.dto';
+import { User } from './model-user';
+import { UsersService } from './users.service';
+
+@Controller('users')
+export class UsersController {
+  constructor (private usersService:UsersService){}
+
+  @Get()
+  getAllUsers(): User[]{
```

Comment:
Insteado of customer entity, you should create another dto for response the user objeto without the password for example. The new dto could be named UserResponseDto.

Reply:
The same apply to the get customer by id.